logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -3,7 +3,7 @@ package com.wire.kalium.logic.sync
 import com.wire.kalium.cryptography.CryptoClientId
 import com.wire.kalium.cryptography.CryptoSessionId
 import com.wire.kalium.cryptography.ProteusClient
-import com.wire.kalium.cryptography.UserId
+import com.wire.kalium.cryptography.PlainUserId
```

Comment:
I think we can remove the alias now that it was renamed to `PlainUserId`

Reply:
There's no alias here.